frontend/src/components/NewAgreement/AgreementCreatedSuccessfully.tsx

```diff
@@ -1,27 +1,90 @@
-import { Box, Button, Container, Divider, Stack, Typography } from '@mui/material';
+import {
+  Box,
+  Button,
+  Container,
+  Divider,
+  IconButton,
+  Stack,
+  SvgIcon,
+  Typography,
+} from '@mui/material';
 import { FC, useContext } from 'react';
 import { useTranslation } from 'react-i18next';
-import DoneRoundedIcon from '@mui/icons-material/DoneRounded';
 import { StringBank } from 'strings';
 import { useNavigate } from 'react-router-dom';
 import { GroupContext } from 'contexts/group';
+import handLogo from 'assets/icons/hand-v@3x.png';
+import styled from 'styled-components';
+import { ReactComponent as CopyIcon } from 'assets/icons/copy.svg';
 
-const AgreementCreatedSuccessfully: FC = () => {
+const CopyIconWrapper = styled(SvgIcon)`
+  svg path {
+    fill: #adb2b8;
+  }
+`;
```

Comment:
No need for this styling, this should be achievable using `<SvgIcon htmlColor="#adb2b8"`/>

Reply:
its not working for me when I write:
const CopyEL: FC<{ text: string }> = ({ text }) => {
  return (
    <IconButton
      sx={{ marginLeft: '2rem', marginRight: '1rem' }}
      onClick={() => {
        navigator.clipboard.writeText(text);
      }}
    >
      <SvgIcon component={CopyIcon} htmlColor="red" />
    </IconButton>
  );
};